templatestore/models.py

```diff
@@ -59,7 +60,8 @@ class TemplateVersion(models.Model):
     deleted_on = models.DateTimeField(null=True, blank=True)
     version_alias = models.CharField(blank=True, max_length=100)
     created_by = models.IntegerField(null=True, blank=True)
-
+    user_email = models.CharField(max_length=200, null=True, blank=True)
```

Comment:
why have we created a new field? we can use created_by field. After saml created_by will always be null so no point keeping a separate field for storing that so lets reuse it for storing email.

Reply:
we can write a migration file to update the exisiting created_by and then we can use it to store emails instead of user_id.

Reply:
yes thats the idea, breaking it into two steps: one add a new field and migrate old values to new field, second to drop old field and rename new field to created_by

if directly modifying the current field, existing values which are stored in created_by field will get lost, also if something breaks we can easily revert back (since field migration would be irreversible)